bpf/process/types/basic.h

```diff
@@ -1686,6 +1686,7 @@ selector_arg_offset(__u8 *f, struct msg_generic_kprobe *e, __u32 selidx,
 			args += 4;
 		case file_ty:
 		case path_ty:
+		case linux_binprm_type:
```

Comment:
This is great, but to match up with the reading of the args, please can you wrap this single line in:
```
#ifdef __LARGE_BPF_PROG

#endif
```
So we only try to match if we've actually read the arg. Otherwise, thanks!